comps/llms/README.md

```diff
@@ -6,111 +6,123 @@ A prerequisite for using this microservice is that users must have a LLM text ge
 
-#### 1.3.1 Verify the TGI Service
+You can use either a published docker image or build your own docker image with the respective microservice Dockerfile of your choice.
 
-```bash
-curl http://${your_ip}:8008/generate \
-  -X POST \
-  -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":17, "do_sample": true}}' \
-  -H 'Content-Type: application/json'
-```
+### 2.1 Start LLM Service with published image
 
-#### 1.3.2 Verify the vLLM Service
+#### 2.1.1 Start TGI Service
 
 ```bash
-curl http://${your_ip}:8008/v1/completions \
-  -H "Content-Type: application/json" \
-  -d '{
-  "model": ${your_hf_llm_model},
-  "prompt": "What is Deep Learning?",
-  "max_tokens": 32,
-  "temperature": 0
-  }'
-```
-
-#### 1.3.3 Verify the Ray Service
+export HF_LLM_MODEL=${your_hf_llm_model}
+export HF_TOKEN=${your_hf_api_token}
 
-```bash
-curl http://${your_ip}:8008/v1/chat/completions \
-  -H "Content-Type: application/json" \
-  -d '{
-  "model": ${your_hf_llm_model},
-  "messages": [
-        {"role": "assistant", "content": "You are a helpful assistant."},
-        {"role": "user", "content": "What is Deep Learning?"},
-    ],
-  "max_tokens": 32,
-  "stream": True
-  }'
+docker run \
+  -p 8008:80 \
+  -e HF_TOKEN=${HF_TOKEN} \
+  -v ./data:/data \
+  --name tgi_service \
+  --shm-size 1g \
+  ghcr.io/huggingface/text-generation-inference:1.4 \
+  --model-id ${HF_LLM_MODEL}
 ```
 
-### 1.4 Start LLM Service with Python Script
-
-#### 1.4.1 Start the TGI Service
+#### 2.1.2 Start vLLM Service
 
 ```bash
-export TGI_LLM_ENDPOINT="http://${your_ip}:8008"
-python text-generation/tgi/llm.py
-```
+# Use the script to build the docker image as opea/vllm:cpu
+bash ${OPEA_GENAICOMPS_ROOT}/comps/llms/text-generation/vllm/build_docker_vllm.sh cpu
 
-#### 1.4.2 Start the vLLM Service
+export HF_LLM_MODEL=${your_hf_llm_model}
+export HF_TOKEN=${your_hf_api_token}
 
-```bash
-export vLLM_LLM_ENDPOINT="http://${your_ip}:8008"
-python text-generation/vllm/llm.py
+docker run -it \
+  --name vllm_service \
+  -p 8008:80 \
+  -e HF_TOKEN=${HF_TOKEN} \
+  -e VLLM_CPU_KVCACHE_SPACE=40 \
+  -v ./data:/data \
+  opea/vllm:cpu \
+  --model ${HF_LLM_MODEL}
+  --port 80
 ```
 
-#### 1.4.3 Start the Ray Service
+#### 2.1.3 Start Ray Service
 
 ```bash
-export RAY_Serve_ENDPOINT="http://${your_ip}:8008"
-python text-generation/ray_serve/llm.py
+export HF_LLM_MODEL=${your_hf_llm_model}
+export HF_CHAT_PROCESSOR=${your_hf_chatprocessor}
+export HF_TOKEN=${your_hf_api_token}
+export TRUST_REMOTE_CODE=True
+
+docker run -it \
+  --runtime=habana \
+  --name ray_serve_service \
+  -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+  --cap-add=sys_nice \
+  --ipc=host \
+  -p 8008:80 \
+  -e HF_TOKEN=$HF_TOKEN \
+  -e TRUST_REMOTE_CODE=$TRUST_REMOTE_CODE \
+  opea/llm-ray:latest \
+  /bin/bash -c "ray start --head && python api_server_openai.py --port_number 80 --model_id_or_path ${HF_LLM_MODEL} --chat_processor ${HF_CHAT_PROCESSOR}"
```

Comment:
Since you're breaking up some of these long lines, how about this one too:
```suggestion
  /bin/bash -c "ray start --head && python api_server_openai.py --port_number 80 --model_id_or_path ${HF_LLM_MODEL} \
     --chat_processor ${HF_CHAT_PROCESSOR}"
```